lib/manager/travis/extract.spec.ts

```diff
@@ -2,21 +2,86 @@ import { loadFixture } from '../../../test/util';
 import { extractPackageFile } from './extract';
 
 const invalidYAML = loadFixture('invalid.yml');
+const matrixYAML = loadFixture('matrix.yml');
+const matrixYAML2 = loadFixture('matrix2.yml');
+const matrixYAML3 = loadFixture('matrix3.yml');
```

Comment:
How about using names like these for the fixture constants?

- `matrixYAMLwithNodeSyntaxString`
- `matrixYAMLwithNodeSyntaxArray`
- `matrixYAMLwithNodeSyntaxAlias`